### PR TITLE
Added jset() and set_stage() functions :D

### PR DESCRIPTION
**Pull request by abdulrabiu**
All /management functions have now been filled!!! \o/ \o/

### CODE REVIEW

File: management/dynamic.py [hunk 1]
```diff
@@ -17,7 +17,10 @@ def jset(variable,value):
     Keyword arguments:
     variable -> the variable that should be changed.  
     value -> the value the variable should be set to."""
-
+    
+    jdict1 = json.loads(open(dynamic_config).read())
+    jdict1[variable] = value
```
Comment (Bentechy66) [inline management/dynamic.py]: Does this write back to disk?

Comment (abdulrabiu) [reply management/dynamic.py]: erm.. I might have forgotten that bit
:p

Comment (abdulrabiu) [reply management/dynamic.py]: I'll be right back

**Comment by abdulrabiu** [comment]
Again - thought I had to make a new pull request. I'm not that familiar with GitHub lol

**Comment by Randium** [comment]
Awesome work! Let's merge this!

**Pull request merged**